plugins/attach.py

```diff
@@ -0,0 +1,19 @@
+# Author: Fayas (https://github.com/FayasNoushad) (@FayasNoushad)
+
+from userge import userge, Message
+
+
+@userge.on_cmd("attach", about={
+    'header': "Attach any link in a message",
+    'usage': "{tr}attach [reply to a message]"})
+async def attach(update: Message):
+    '''Attach links in message''' 
+    if update.reply_to_message is None:
+        await update.reply_to_message.reply_text(
+            text="Reply to a text for attaching"
+        )
+        return
+    message = update.reply_to_message
+    link = message.text.split(" ", 1)[1]
```

Comment:
this should `update.text.split...` right ?

Reply:
Like `. attach <link>` replying a message.

Reply:
yes, but you have defined `message = update.reply_to_message` in above line